com.unity.render-pipelines.universal/Shaders/BakedLitInput.hlsl

```diff
@@ -4,21 +4,24 @@
 #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/SurfaceInput.hlsl"
 
 CBUFFER_START(UnityPerMaterial)
-float4 _BaseMap_ST;
-half4 _BaseColor;
-half _Cutoff;
-half _Glossiness;
-half _Metallic;
+    half _Surface;
```

Comment:
I'd expect that if you add this at the end you get better padding/layout.

Reply:
Another suggestion `_SurfaceType`

Reply:
That would require a material upgrader or having two different parameters for the same thing in our shaders.